applications/electron/test/app.spec.js

```diff
@@ -27,7 +27,7 @@ function getBinaryPath() {
         "TheiaBlueprint.app",
         "Contents",
         "MacOS",
-        "TheiaBlueprint"
+        "blueprint-electron-app"
```

Comment:
We should revert the change in order for the `macOS` tests to pass:

```suggestion
        "TheiaBlueprint"
```

Reply:
done, thanks Vince!